dev-tools/iota-data-ingestion/README.md

```diff
@@ -26,14 +26,12 @@ path: "./test-checkpoints"
 # IOTA Node Rest API URL
 remote-store-url: "http://localhost:9000/api/v1"
 
-# DynamoDbProgressStore config
+# Path to the progress store JSON file.
```

Comment:
Is the Environment Variables section (starting from line 7) still needed?

Reply:
@jkrvivian yes is till needed because internally the `iota-data-ingestion` crate has workers which internally store checkpoint information to `DynamoDB` and `S3`